src/easystar.js

```diff
@@ -421,7 +435,11 @@ EasyStar.js = function() {
     };
 
     // Helpers
-    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y) {
+    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y, sourceNode) {
+        if (directionalConditions[x + "_" + y]) {
+            var direction = calculateDirection(sourceNode.x - x, sourceNode.y - y)
+            if (!includes(directionalConditions[x + "_" + y], direction)) return false
```

Comment:
That seems good to me. In the jsdoc it says direction can be a string or an array. Does this mean it must now be an array ?